mypy.ini

```diff
@@ -12,6 +12,13 @@ exclude = examples, test
 
 python_version = 3.7
 
+#
+# Extension modules without stubs.
+#
+
+[mypy-torchdata._torchdata]
```

Comment:
Should this be in a different format? @ejguan

Reply:
My bad. Let me push a commit to your branch.

Reply:
Are you working on pushing right now?

For a quick fix on python version, starting from CMake 3.19, we can specify versions directly. https://cmake.org/cmake/help/latest/command/find_package.html#id5
> The [version] argument requests a version with which the package found should be compatible. There are two possible forms in which it may be specified:
> 
> A single version with the format major[.minor[.patch[.tweak]]], where each component is a numeric value.